### ISSUE TITLE
⌨  Skills library missing lang attribute

### ISSUE THREAD

**Issue by esizer** [GCTC-NTGC/gc-digital-talent#17684]
### ⌨ Accessibility and Expected Behaviour

The skills library contains both French and English text but does not denote that. This means, some cells have languages that differ from the main site language.

When this happens, we need to add the `lang` atgtribute so that assistive technologies are aware of the change.

### WCAG 2.1 Criteria

 - https://www.w3.org/WAI/WCAG22/Techniques/html/H58
 - https://www.w3.org/WAI/WCAG22/Understanding/language-of-parts

### 🙋‍♀️ Proposed Solution

Update the skills table to apply `lang` attrbiutge to the cells that differ from the main page language code. i.e.

 - When the page is in English, the French cells should have `lang="fr"`
 - When the page is in Frenhc, the English cells should have `lang="en"`

### ✅ Acceptance Criteria

- [ ] Cells in the skills livbrary that differ from the main page language have the appropriate lang attribute
